deid/utils/actions.py

```diff
@@ -141,3 +160,41 @@ def get_timestamp(item_date, item_time=None, jitter_days=None, format=None):
         timestamp = timestamp + timedelta(days=jitter_days)
 
     return timestamp.strftime(format)
+
+
+def convert_value(field, VR, value):
+    """
+    convert_value converts the value specified into the appropriate Python types
+    for writing by pydicom.
+    https://pydicom.github.io/pydicom/dev/guides/element_value_types.html
+
+    If the value cannot be casted to the appropriate type, it is converted to None
+    and will be blanked by the operation.
+    """
+
+    if VR in ["FL", "FD"]:
+        try:
+            return float(value)
+        except (ValueError, TypeError):
+            bot.warning(
+                f"Value ({value}) is not a valid value for VR {VR} field: {field}. Field will be BLANKED."
+            )
+            return None
+    elif VR in ["OB", "OD", "OF", "OL", "OV", "OW", "UN"]:
+        try:
+            return bytes(value, "utf-8")
+        except (ValueError, TypeError):
+            bot.warning(
+                f"Value ({value}) is not a valid value for VR {VR} field: {field}. Field will be BLANKED."
+            )
+            return None
+    elif VR in ["SL", "SS", "SV", "UL", "US", "UV"]:
+        try:
+            return int(value)
+        except (ValueError, TypeError):
+            bot.warning(
+                f"Value ({value}) is not a valid value for VR {VR} field: {field}. Field will be BLANKED."
+            )
+            return None
+    else:
```

Comment:
I don't think we need the else - if we reach the end of the function, just return value, as all the checks above return something.